composer.json

```diff
@@ -21,7 +21,8 @@
         "symfony-cmf/testing": "^1.3@dev",
         "symfony/dependency-injection": "^2.0.5|3.*",
         "symfony/config": "^2.2|3.*",
-        "symfony/event-dispatcher": "^2.1|3.*"
+        "symfony/event-dispatcher": "^2.1|3.*",
+        "friendsofsymfony/jsrouting-bundle": "@dev"
```

Comment:
okay, i see. i hope to release this library really soon - should we do so with this line or should we try to tag a stable version of jsrouting?